starter-code/functions-and-arrays.js

```diff
@@ -61,9 +99,26 @@ var words = [
 var averageLength = averageWordLength(words);
 console.log(averageLength);
 
+
+
+// ABOVE ALL SEEMS TO BE CORRECT
+
+
+
 // Unique Arrays
+
 function uniquifyArray (array) {
```

Comment:
You are trying to remove duplicates from the array. This approach is very hard. An easier one would be to create a new array in which you add elements you haven't seen before.